CIP-0129? | Governance Identifiers (#857)
cardano-foundation/CIPs: CIP-0105/README.md

| `drep_vk`     | CIP-1852’s DRep verification key                        | Ed25519 public key                                                |
| `drep_xsk`    | CIP-1852’s DRep extended signing key                    | Ed25519-bip32 extended private key                                |
| `drep_xvk`    | CIP-1852’s DRep extended verification key               | Ed25519 public key with chain code                                |
| `drep_vkh`    | Delegate representative verification key hash           | blake2b\_224 digest of a delegate representative verification key |

> | `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |

Reviewer: ```suggestion
| `drep` | Deprecated.... | |
| `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |
```

> | `drep_vkh`    | Delegate representative verification key hash           | blake2b\_224 digest of a delegate representative verification key |
> | `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |

Reviewer 2: ```suggestion
| `drep` | [DEPRECATED] Delegate representative verification key hash (DRep ID) | blake2b\_224 digest of a delegate representative verification key |
| `drep_vkh`    | Delegate representative verification key hash           | blake2b\_224 digest of a delegate representative verification key |
| `drep_script` | Delegate representative script hash                     | blake2b\_224 digest of a serialized delegate representative script |
```

Reviewer: Might be worth referencing CIP-129 where we say depreciated

Author: the deprecated tag links to the deprecation section

| `cc_cold_vk`     | CIP-1852’s constitutional committee verification signing key          | Ed25519 private key                                                    |
| `cc_cold_xsk`    | CIP-1852’s constitutional committee cold extended signing key         | Ed25519-bip32 extended private key                                     |
| `cc_cold_xvk`    | CIP-1852’s constitutional committee extended verification signing key | Ed25519 public key with chain code                                     |

> | `cc_cold_vkh`    | Constitutional committee cold verification key hash (cold credential) | blake2b\_224 digest of a consitutional committee cold verification key |

Reviewer 2: ```suggestion
| `cc_cold`        | [DEPRECATED] Constitutional committee cold verification key hash (cold credential) | blake2b\_224 digest of a consitutional committee cold verification key |
| `cc_cold_vkh`    | Constitutional committee cold verification key hash (cold credential) | blake2b\_224 digest of a consitutional committee cold verification key |
```

Author: updated

| `cc_hot_vk`     | CIP-1852’s constitutional committee verification signing key          | Ed25519 private key                                                   |
| `cc_hot_xsk`    | CIP-1852’s constitutional committee hot extended signing key          | Ed25519-bip32 extended private key                                    |
| `cc_hot_xvk`    | CIP-1852’s constitutional committee extended verification signing key | Ed25519 public key with chain code                                    |

> | `cc_hot_vkh`    | Constitutional committee hot verification key hash (hot credential)   | blake2b\_224 digest of a consitutional committee hot verification key |

Reviewer 2: ```suggestion
| `cc_hot`        | [DEPRECATED] Constitutional committee hot verification key hash (hot credential)   | blake2b\_224 digest of a consitutional committee hot verification key |
| `cc_hot_vkh`    | Constitutional committee hot verification key hash (hot credential)   | blake2b\_224 digest of a consitutional committee hot verification key |
```

Author: updated

| `ConstitutionalCommitteeHotHWSigningFile_ed25519`   | Hardware Constitutional Committee Hot Signing File     |
| `ConstitutionalCommitteeHotVerificationKey_ed25519` | Hardware Constitutional Committee Hot Verification Key |

### Deprecated Governance ID Definition

> The previous governance key IDs defined by this have been upgraded as per the new specification introduced in [CIP-0129]. Tools implementing the following specification should gradually adopt the updated ID format outlined in [CIP-0129]. Tools that already support [CIP-0129] maintain backward compatibility with the legacy formats specified below but should consider fully transitioning to [CIP-0129] to standardize key formats across the ecosystem. This will help avoid multiple formats and ensure consistency.

Reviewer: ```suggestion
The previous governance key IDs defined by this standard have been superseded by the definitions provided in [CIP-0129]. Tools implementing this standard are encouraged to consider adopting [CIP-0129]. Tools that already support [CIP-0129] maintain backward compatibility with the legacy formats specified below but should consider fully transitioning to [CIP-0129] to standardize key formats across the ecosystem. This will help avoid multiple formats and ensure consistency.
```
tweaks on wording

Author: done
